### ISSUE TITLE
Can we add one more additional date format yMMMM?

### ISSUE THREAD

**Issue by ZxMYS** [twitter/twitter-cldr-js#69]
I know we have yMMM, but being able to display full month name when I don't have day is handy.
Thanks!

**Comment by camertron** [comment]
Hey @ZxMYS do you know if such a format is supported by CLDR?

**Comment by ZxMYS** [comment]
@camertron yes I think so, it's used as an example on http://cldr.unicode.org/translation/date-time-patterns#TOC-Additional-Date-Time-Formats, and https://github.com/twitter/twitter-cldr-rb/search?utf8=%E2%9C%93&q=yMMMM contains many results.

**Comment by camertron** [comment]
Ah ok, thanks for doing that research. My guess is this additional format isn't available in twitter-cldr-js because we haven't updated to the most recent version of CLDR. Thanks for filing this as an issue, I'll take a look this weekend.

**Comment by ZxMYS** [comment]
Thanks @camertron !

**Comment by camertron** [comment]
I didn't have time to look at this over the weekend, but my hunch is that it's going to take some effort. We'll need to bring twitter-cldr-rb up to speed with the latest CLDR release, which could prove to be challenging depending how many things have changed between releases. Then we'll have to port any code and data changes to twitter-cldr-js. My guess is we won't have such work done for a few months at least. Upgrading to v27 (the latest, I believe) is definitely on our radar for twitter-cldr-rb.

**Comment by web-padawan** [comment]
@camertron the latest CLDR release is now v29. Are there any plans for upgrading?

**Comment by camertron** [comment]
@web-padawan yes, but only as time permits. I don't have much time to contribute to this lib or twitter-cldr-rb at the moment. Upgrading CLDR versions is on my/our radar however. Naturally we would welcome any pull requests you and the community have the time to contribute!
